tests/PHPStan/Analyser/AnalyserIntegrationTest.php

```diff
@@ -513,6 +513,16 @@ public function testBug6253(): void
 		$this->assertNoErrors($errors);
 	}
 
+	public function testBug6442(): void
+	{
+		$errors = $this->runAnalyse(__DIR__ . '/data/bug-6442.php');
+		$this->assertCount(2, $errors);
+		$this->assertSame('Dumped type: \'Bug6442\\\A\'', $errors[0]->getMessage());
```

Comment:
You can use `assertType()` instead and still assert no errors.

Reply:
sorry, where would I do that? in the test file won't work I guess, because the same asserType would be called twice with different expected types  :/

Reply:
Oh, alright!